packages/markdown/src/components/Code.tsx

```diff
@@ -0,0 +1,26 @@
+import { List } from "@alloy-js/core";
+import { Children } from "@alloy-js/core/jsx-runtime";
+
+export interface CodeProps {
+  /** Language of the code block */
+  lang?: string;
+
+  /** Content of the codeblock, use instead of children. */
+  content?: string;
+  /** Body of the code block */
+  children?: Children;
+}
+
+/**
+ * Render a Markdown code block
+ */
+export function Code(props: CodeProps) {
+  const tripleBackticks = "```";
+  return (
+    <List>
+      {tripleBackticks + props.lang}
+      {props.content ?? <List children={props.children} />}
```

Comment:
I think we don't want the List for the `Code` component because we don't want to mess up formatting of the code contained therein. I think drop the content prop and delegate all formatting to the Children.